### PR TITLE
Add volunteers page

### PR DESCRIPTION
**Pull request by sabderemane**

### CODE REVIEW

File: _pages/volunteers.html [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+      <hr>
+
+     <h3>Social</h3>
+
+      <ul>
+        <li>Main social event:</li>
+        <ul>
+          <li>Checking tickets at the door</li>
+          <li>leading activities (board games, crafts, sticker swaps, etc.)</li>
```
Comment (cgl) [inline _pages/volunteers.html:97]:
```suggestion
          <li>Leading activities (board games, crafts, sticker swaps, etc.)</li>
```

File: _pages/volunteers.html [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+      <hr>
+
+      <h3>Setup/teardown</h3>
+       <ul>
+        <li>The Assembly Rooms (main conference venue):</li>
+        <ul>
+          <li>Sunday 28th May: setup 6-10pm</li>
+          <li>Wednesday 31st May: teardown 5-8pm</li>
+        </ul>
+        <li>The Biscuit factory (sprints)</li>
```
Comment (cgl) [inline _pages/volunteers.html:111]:
```suggestion
        <li>The Biscuit factory (sprints):</li>
```

File: _pages/volunteers.html [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+         <ul>
+          <li>Wednesday 31st May: setup 5-8pm (TBC)</li>
+          <li>Thursday 1st June: setup 8-10am (TBC)</li>
+          <li>Friday 2nd June:5-7pm teardown</li>
+        </ul>
+      </ul>
+      <p>This job involves walking and carrying items</p>
```
Comment (cgl) [inline _pages/volunteers.html:118]:
```suggestion
      <p>This job involves walking and carrying items.</p>
```

**Review by cgl — APPROVED** [review]
Looks good to me, suggested minor changes to improve the formatting and wording.

File: _pages/volunteers.html [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+      <ul>
+        <li>It’s a chance to meet new people and possibly make friends with people you may not have met otherwise. If you’re new to DjangoCon Europe, volunteering is a great way to connect with other attendees.</li>
+        <li>It’s a great opportunity to discover a career path: volunteering provides individuals with new experiences and skills</li>
+        <li>Session chairing is a great way to get to meet the speakers!</li>
+        <li>You’ll get an insight into what’s involved with running the conference and contribute to the team!</li>
+      </ul>
+
+      <h2 class="v-pad-top">How to volunteer</h2>
+
+      <p>Register your interest in volunteering by completing <a href="https://forms.gle/xpdbLLrA7ps6Erjy9">this form</a>. We’ll be in touch nearer the time with more information.</p>
```
Comment (rebkwok) [inline _pages/volunteers.html:41]:
```suggestion
      <p>Register your interest in volunteering by completing <a href="https://forms.gle/xpdbLLrA7ps6Erjy9">the registration form</a>. We’ll be in touch nearer the time with more information.</p>
```

**Review by rebkwok — COMMENTED** [review]
LGTM (with one suggestion, and @cgl's changes)

**Pull request merged**